src/main/java/com/ibm/cloudant/kafka/schema/ConnectRecordMapper.java

```diff
@@ -100,4 +124,12 @@ private Object getField(Type type, Object value) {
             
     }
 
+    private String getHeaderForDocId(ConnectRecord<R> record) {
+        Header value = record.headers().lastWithName(HEADER_DOC_ID_KEY);
+        if (value != null && value.value() instanceof String) {
+            return value.value().toString();
```

Comment:
Could also be a cast to `String` but it's a style issue as they both achieve the same thing.

Reply:
I'd prefer to keep this as-is